v0.1/reference_submissions/person_detection/parse_coco.py

```diff
@@ -0,0 +1,174 @@
+# Lint as: python3
+'''Extract the MS COCO dataset into person/not person training dataset.
```

Comment:
is there a static version of the VWW? If not should/can we host one? Extracting it every time seems like it could be a source of error.

@jveejay Are we able to host a dataset?